saleor/product/models.py

```diff
@@ -162,14 +163,24 @@ def collection_sorted(self, user: "User"):
         )
         return qs
 
-    def sort_by_attribute(self, attribute_pk: Union[int, str], ascending: bool = True):
+    def sort_by_attribute(
+        self, attribute_pk: Union[int, str], descending: bool = False
+    ):
         """Sort a query set by the values of the given product attribute.
 
         :param attribute_pk: The database ID (must be a number) of the attribute
                              to sort by.
```

Comment:
This description doesn't fit to typing in method definition :information_desk_person:

Reply:
fixed